pallets/portfolio/src/lib.rs

```diff
@@ -143,6 +163,14 @@ decl_error! {
         DestinationIsSamePortfolio,
         /// The portfolio couldn't be renamed because the chosen name is already in use.
         PortfolioNameAlreadyInUse,
+        /// The porfolio's custody is with someone other than the caller.
+        UnauthorizedCustodian,
+        /// The authorization is for something other than portfolio custody
+        IrrelevantAuthorization,
+        /// Can not unlock more tokens than what are locked
+        InsufficientTokensLocked,
+        /// Transfers from and to the portfolio are not allowed
+        SelfTransferNotAllowed
```

Comment:
Duplicate of `DestinationIsSamePortfolio`. Apart from that, the new term "self-transfer" isn't informative.